astropy/modeling/projections.py

```diff
@@ -840,19 +821,20 @@ class Pix2Sky_CylindricalEqualArea(Pix2SkyProjection, Cylindrical):
     lam : float
         Radius of the cylinder in spherical radii, λ.  Default is 1.
     """
+    prj_code = 'CEA'
 
     lam = Parameter(default=1)
 
+    @lam.validator
+    def lam(self, value):
+        self._dirty = True
+
     @property
     def inverse(self):
         return Sky2Pix_CylindricalEqualArea(self.lam)
 
-    @classmethod
-    def evaluate(cls, x, y, lam):
-        return _projections.ceax2s(x, y, lam)
-
-
-Pix2Sky_CEA = Pix2Sky_CylindricalEqualArea
+    def _update_prj(self):
+        self._prj.set_pvi(1, float(self.lam.value))
```

Comment:
Because this is one of the few projections that has only one parameter and it is more efficient to use a function that is specialized in modifying a single value.

Reply:
If efficiency is a minor concern (I think it is here), consistency is preferable.

Reply:
Let's compare two calls:
```python
self._prj.set_pvi(1, float(self.lam.value))
self._prj.set_pv([None, float(self.lam.value)])
```
Is second call more clear or obvious about what it is doing?

Reply:
Neither. It is a consistency issue which is orthogonal to the clarity issue. And may be besides the point if you take my more meta suggestion.

Reply:
> meta suggestion

NOOOO!